.dockerignore

```diff
@@ -4,6 +4,5 @@
 /node_modules
 __pycache__/
 
-/example_docs/docs/_build/
```

Comment:
I added this for the sake of docs previews. It wasn't totally necessary, only "more clean". But it caused the Dockerfile for snapshot tests to fail when run locally because it couldn't discover this folder it tries to copy.